defaults/main.yml

```diff
@@ -18,6 +18,10 @@ wimpy_application_pre_commands: []
 wimpy_application_post_commands: []
 wimpy_application_additional_units: []
 
+# CloudWatch logs
+wimpy_aws_cloudwatch_enabled: True
```

Comment:
Wep,

I agree to enabled it by default, perhaps it should be documented on readme file.